IETF-RFC.md

```diff
@@ -1394,6 +1438,337 @@ format:
 }
 ```
 
+
+
+# Appendix D: Object models
+
+An implementor of OCM MAY choose any internal object model to represent
+an _Address Book_, a _Contact_, an _Invite_, a _Provider_, a _Share_,
+and  a _User_.  However, the following diagrams are provided to clarify
+the concepts and their relationships, as a guide for implementors.
+
+## Address Book
+
+An _OCM Provider_ MAY offer its _Users_ an address book tool, where OCM
+Addresses can be stored over time in a labeled and/or searchable way.
+This decouples the act by which the OCM Address string is passed into
+the Sending Server's database from the selection of the _Receiving
+Party_ in preparation for Share Creation.
+
+The Address Book entity maintains a collection of contacts for a user
+within the OCM provider. It serves as the primary mechanism for managing
```

Comment:
Here and in the following we should still follow the IETF convention of two-spaces-after-full-stop